examples/jsm/loaders/KTX2Loader.js

```diff
@@ -260,13 +260,17 @@ class KTX2Loader extends Loader {
 	 * @return {Promise<CompressedTexture|DataTexture|Data3DTexture>}
 	 */
 	_createTexture( buffer, config = {} ) {
```

Comment:
Perhaps let's make the method async, instead:

```suggestion
	async _createTexture( buffer, config = {} ) {
```

That should have the same effect without requiring extra code. I think there are other things we can simplify later by going this route, too.

Reply:
@jameszhong2008 Would you mind updating the PR according to @donmccurdy's suggestion?